### ISSUE TITLE
Oracle PL/SQL Prototypes not detected

### ISSUE THREAD

**Issue by MrUrdam** [NaturalDocs/NaturalDocs#61]
I can not get the automatic prototype detection for oracle PL/SQL to work.
Below is a code example that includes many different cases. But I can also not get a simple one to work.

``` SQL
/*
 * Procedure: addSomething
 * Description of the method...
 *
 * Parameters:
 * param1 - Parameter1
 * param2 - Parameter2
 *
 * Returns:
 * Returns a value...
 */
FUNCTION addSomething(
  param1 IN VARCHAR2,
  param2 OUT NUMBER DEFAULT 1234,
  param3 IN OUT CLOB,
  param4 IN OUT CHAR := 'abc',
  param5 DATE
) RETURN VARCHAR2
IS
...
```

If i add a manual prototyping everything works like expected.
I have the default language settings, that includes the (in my opinion) correct endings.

I have tested the latest stable and the current development version.

**Comment by MrUrdam** [comment]
My indentation puts two spaces before the keyword PROCEDURE/PACKAGE.
If i remove them, the prototype is detected.
I think that should be a minor issue in your parser ?

**Comment by NaturalDocs** [comment]
I think I fixed it.  Try this version:
[https://www.naturaldocs.org/download/temp/Natural_Docs_temp4.zip](https://www.naturaldocs.org/download/temp/Natural_Docs_temp4.zip)

**Comment by MrUrdam** [comment]
This fixed the issue.
Thank you for your help :)

**Comment by NaturalDocs** [comment]
This fix has been included in Natural Docs 2.1.1 and Natural Docs 2.2 Development Release 3.
